_includes/_readme/credits.md

```diff
@@ -0,0 +1,102 @@
+# Credits
```

Comment:
we might wanna have some documentation somewhere on which files currently are cherry-picked / need to be manually updated.

Reply:
yes, I'll be opening issues once this is merged